cmd/solvercli/cmd/settle_orders.go

```diff
@@ -0,0 +1,153 @@
+
+	lmt.ConfigureLogger()
+	ctx = lmt.LoggerContext(ctx)
+
+	keyStoreType, err := cmd.Flags().GetString("key-store-type")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get key-store-type", zap.Error(err))
+		return
+	}
+
+	keysPath, err := cmd.Flags().GetString("keys")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get keys path", zap.Error(err))
+		return
+	}
+
+	sqliteDBPath, err := cmd.Flags().GetString("sqlite-db-path")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get sqlite-db-path", zap.Error(err))
+		return
+	}
+
+	migrationsPath, err := cmd.Flags().GetString("migrations-path")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get migrations-path", zap.Error(err))
+		return
+	}
+
+	configPath, err := cmd.Flags().GetString("config")
+	if err != nil {
+		lmt.Logger(ctx).Error("Failed to get config path", zap.Error(err))
+		return
+	}
+
+	cfg, err := config.LoadConfig(configPath)
+	if err != nil {
+		lmt.Logger(ctx).Error("Unable to load config", zap.Error(err))
+		return
+	}
+
+	ctx = config.ConfigReaderContext(ctx, config.NewConfigReader(cfg))
+
+	keyStore, err := keys.GetKeyStore(keyStoreType, keys.GetKeyStoreOpts{KeyFilePath: keysPath})
+	if err != nil {
+		lmt.Logger(ctx).Fatal("Unable to load keystore", zap.Error(err))
+	}
+
+	cosmosTxExecutor := cosmos.DefaultSerializedCosmosTxExecutor()
+	evmTxExecutor := evm.DefaultEVMTxExecutor()
+
+	clientManager := clientmanager.NewClientManager(keyStore, cosmosTxExecutor)
+
+	dbConn, err := connect.ConnectAndMigrate(ctx, sqliteDBPath, migrationsPath)
+	if err != nil {
+		lmt.Logger(ctx).Fatal("Unable to connect to db", zap.Error(err))
+	}
+	defer dbConn.Close()
+
+	evmManager := evmrpc.NewEVMRPCClientManager()
+	rateLimitedClient := utils.DefaultRateLimitedHTTPClient(3)
+	coingeckoClient := coingecko.NewCoingeckoClient(rateLimitedClient, "https://api.coingecko.com/api/v3/", "")
+	cachedCoinGeckoClient := coingecko.NewCachedPriceClient(coingeckoClient, 15*time.Minute)
+	evmTxPriceOracle := evmrpc.NewOracle(cachedCoinGeckoClient)
+
+	hype, err := hyperlane.NewMultiClientFromConfig(ctx, evmManager, keyStore, evmTxPriceOracle, evmTxExecutor)
+	if err != nil {
+		lmt.Logger(ctx).Fatal("creating hyperlane multi client from config", zap.Error(err))
+	}
+
+	relayer := hyperlane.NewRelayer(hype, make(map[string]string))
+	relayerRunner := hyperlane.NewRelayerRunner(db.New(dbConn), hype, relayer)
+
+	settler, err := ordersettler.NewOrderSettler(ctx, db.New(dbConn), clientManager, relayerRunner)
+	if err != nil {
+		lmt.Logger(ctx).Error("creating order settler", zap.Error(err))
+		return
+	}
+
+	batches, err := settler.PendingSettlementBatches(ctx)
+	if err != nil {
+		lmt.Logger(ctx).Error("getting pending settlement batches", zap.Error(err))
+		return
+	}
+
+	if len(batches) == 0 {
+		fmt.Println("No pending settlement batches found")
+		return
+	}
+
+	fmt.Printf("Found %d pending settlement batches\n", len(batches))
+
+	hashes, err := settler.SettleBatches(ctx, batches)
```

Comment:
Note that this will initiate settlements but not relay the hyperlane message to the destination. How were you intending for the relay to complete?

Reply:
Added the batch relays

Reply:
The hyperlane validator set usually takes a bit to generate the checkpoints signatures. What happens if those signatures aren't generated here? Were you able to test this?